src/barfTurn.ts

```diff
@@ -237,7 +227,7 @@ const turns: AdventureAction[] = [
         })
       );
       adventureMacroAuto(
-        isGhost ? noWineglassZone() : determineDraggableZoneAndEnsureAccess(),
+        isGhost ? drunkSafeWander("wanderer") : wanderWhere("wanderer"),
```

Comment:
I'm still not 100% sure the meaning of `drunkSafeWander` is clear, and given that it is only used in barfturn, it might make more sense to define it in `barfTurn` instead